dll/win32/shell32/CNewMenu.cpp

```diff
@@ -565,8 +565,10 @@ HRESULT CNewMenu::NewItemByNonCommand(SHELLNEW_ITEM *pItem, LPWSTR wszName,
     }
     else
     {
-        StringCbPrintfW(wszBuf, sizeof(wszBuf), L"Cannot create file: %s", wszName);
-        MessageBoxW(NULL, wszBuf, L"Cannot create file", MB_OK | MB_ICONERROR); // FIXME load localized error msg
+        CStringW m_Caption(MAKEINTRESOURCEW(IDS_CREATEFILE_CAPTION));
+        CStringW m_Message(MAKEINTRESOURCEW(IDS_CREATEFILE_DENIED));
+        m_Message.FormatMessage(m_Message.GetString(), wszName);
+        MessageBoxW(0, m_Message, m_Caption, MB_ICONEXCLAMATION | MB_OK);
```

Comment:
We generally use the `m_` prefix for member variables, which these are not.

Reply:
Thanks